package.json

```diff
@@ -7,7 +7,7 @@
   },
   "main": "index.js",
   "scripts": {
-    "test": "jest && ./test/index.sh",
+    "test": "./test/cleanup.sh && jest && ./test/index.sh",
```

Comment:
@luciotato I think it's best to call `./test/cleanup.sh` from `./test/index.sh`

Reply:
I think the `tmp-project` dir needs to be removed before calling jest
also if we leave it before jest, `./test/cleanup.sh` can be used to group all the test-cleanup tasks, not just fixing this problem.